spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1914,48 +1914,58 @@ Spring Cloud Stream provides a health indicator for binders.
 It is registered under the name of `binders` and can be enabled or disabled by setting the `management.health.binders.enabled` property.
 
 == Metrics Emitter
+
 Spring Cloud Stream provides a module called `spring-cloud-stream-metrics` that can be used to emit any available metric from https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-metrics.html[Spring Boot metrics endpoint] to a named channel.
 This module allow operators to collect metrics from stream applications without relying on polling their endpoints.
 
 HTTP polling can be challenging on cloud environments since applications could be on a private network or behind a Load balancer that prevents per instance access.
 
-The module is activated when you set the destination name for its channel, `spring.cloud.stream.bindings.applicationMetricsChannel.destination=<DESTINATION_NAME>`.
-`applicationMetricsChannel` can be configured in a similar fashion to any other bound output channel.
-The default `contentType` setting of `applicationMetricsChannel` is `application/json`.
-
-By default the module is configured to only send Spring Integration message channel metrics.
+The module is activated when you set the destination name for metrics binding, e.g. `spring.cloud.stream.bindings.applicationMetrics.destination=<DESTINATION_NAME>`.
+`applicationMetrics` can be configured in a similar fashion to any other producer binding.
+The default `contentType` setting of `applicationMetrics` is `application/json`.
 
-Available properties for customization using the prefix `spring.cloud.stream.metrics`.
+The following properties can be used for customizing the emission of metrics:
 
-key::
+spring.cloud.stream.metrics.key::
   The name of the metric being emitted. Should be an unique value per application.
 +
 Default:: `${spring.application.name:${vcap.application.name:${spring.config.name:application}}}`
 +
-delay-millis::
-  The period in which metrics will be posted to the channel
-+
-Default: `5000`.
-+
-prefix::
+spring.cloud.stream.metrics.prefix::
   Prefix string to be prepended to the metrics name
 +
 Default: ``
 +
+spring.cloud.stream.metrics.properties::
+  Just like the `includes` option, it allows white listing application properties that will be added to the metrics payload
++
+Default: null.
++
+
+Spring Cloud Stream provides a metric exporter named `application` that can be configured via regular Spring Boot metrics configuration properties.
+A detailed overview can be found in the https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-metrics.html#production-ready-metric-writers[Spring Boot reference documentation].
+
+Summarizing, the exporter can be configured either by using the global Spring Boot configuration settings for exporters, or by using exporter-specific properties.
+A subset of the available properties is listed below.
```

Comment:
I'd remove this sentence and just have the other two usage example say,  "for example, you can configure blah blah..."